common/text/text_structure.h

```diff
@@ -165,8 +169,19 @@ class TextStructureView {
   // TokenInfo::right() to calculate byte offsets, useful for diagnostics.
   absl::string_view contents_;
 
-  // Line-by-line view of contents_.
-  std::vector<absl::string_view> lines_;
+  struct LineInfo {
```

Comment:
suggest naming `LinesInfo`, to suggest that this is concerns multiple lines.

Reply:
Done.